spec/features/create_account_spec.rb

```diff
@@ -21,42 +21,39 @@
       expect(page).to have_content('6 characters minimum')
     end
 
-    scenario 'specifying valid information' do
-      visit new_user_registration_path
-      fill_in 'Email', with: 'Koala@gmail.com'
-      fill_in 'Password', with: 'password'
-      fill_in 'Password confirmation', with: 'password'
-      click_button 'Sign up'
-      expect(page).to have_content("Welcome")
-    end
-
     scenario 'passwords do not match' do
       visit new_user_registration_path
       fill_in 'Email', with: 'X12@gmail.com'
+      fill_in 'Username', with: 'X12'
       fill_in 'Password', with: 'password'
       fill_in 'Password confirmation', with: 'psassword'
       click_button 'Sign up'
       expect(page).to have_content("Password confirmation doesn't match Password")
     end
 
 
-    scenario 'user name is not unique' do
+    scenario 'user email or username is not unique fails to create new user' do
       old_user = User.create(
         email: 'JoeShmo@gmail.com',
```

Comment:
I'd really like to see you using FactoryGirl (and a `User` factory) here instead.